src/_includes/templates/forms/accessibility-feedback-form.html

```diff
@@ -0,0 +1,201 @@
+                   id="accessibility-form_requestor-address-2"
+                   type="text">
+        </div>
+
+        <div class="form-l_col form-l_col-1-3">
+            <label class="form-label-header"
+                   for="accessibility-form_requestor-city">
+                City*
+            </label>
+            <input class="input__long"
+                   id="accessibility-form_requestor-city"
+                   type="text"
+                   required>
+        </div>
+
+        <div class="form-l_col form-l_col-1-3">
+            <label class="form-label-header"
+                   for="accessibility-form_requestor-state">
+                State*
+            </label>
+            <div class="custom-select">
+                <select class="custom-select_select"
+                        data-placeholder="Select state"
+                        id="accessibility-form_requestor-state"
+                        required>
+                    <option value="">Select state</option>
+                    <option value="AL">Alabama</option>
+                    <option value="AK">Alaska</option>
+                    <option value="AZ">Arizona</option>
+                    {# TODO: Add all states to select list. #}
+                    <option value="Other">Other States to be added</option>
+                </select>
+                <span class="custom-select_icon
+                             cf-icon
+                             cf-icon-down"></span>
+            </div>
+        </div>
+        <div class="form-l_col form-l_col-1-3">
+            <label class="form-label-header"
+                   for="accessibility-form_requestor-zip">
+                Zip code*
+            </label>
+            <input class="input__long"
+                   id="accessibility-form_requestor-zip"
+                   type="text"
+                   required>
+        </div>
+
+        <div class="form-l_col form-l_col-1-2">
+            <label class="form-label-header"
+                   for="accessibility-form_telephone-number">
+                Telephone number*
+            </label>
+            <input class="input__long"
+                   id="accessibility-form_telephone-number"
+                   type="number"
+                   required>
+        </div>
+
+        <div class="form-l_col form-l_col-1-2">
+            <label class="form-label-header"
+                   for="accessibility-form_fax-number">
+                Fax number
+            </label>
+            <input class="input__long"
+                   id="accessibility-form_fax-number"
+                   type="text">
+        </div>
+
+        <div class="form-l_col form-l_col-1-2">
+            <label class="form-label-header"
+                   for="accessibility-form_email-address">
+                Email address
+            </label>
+            <input class="input__long"
+                   id="accessibility-form_email-address"
+                   type="text">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/d0dda373002cbc979e937556fcb353f58735775b